requirements/edx/base.txt

```diff
@@ -52,7 +52,7 @@ edx-lint==0.4.3
 astroid==1.3.8
 edx-django-oauth2-provider==1.1.4
 edx-django-sites-extensions==2.1.1
-edx-enterprise==0.36.3
```

Comment:
this won't be included in the final changes - i just have it here because i wanted to have both of my features on the sandbox.